crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -386,7 +407,44 @@ pub(crate) fn orig_name(n: &ModuleExportName) -> JsWord {
     }
 }
 
-fn get_import_symbol_from_import(specifier: &ImportSpecifier) -> ImportedSymbol {
+fn parse_with(with: Option<&ObjectLit>) -> Option<ImportedSymbol> {
+    let with = with?;
+
+    for prop in &with.props {
+        if let PropOrSpread::Prop(prop) = prop {
+            if let Prop::KeyValue(KeyValueProp {
+                key: PropName::Ident(key),
+                value,
+            }) = &**prop
+            {
+                if key.sym == "__turbopack_chunk__" {
```

Comment:
Could we rename that from `chunk` to `part`?

Chunk has a specific meaning in turbopack and I don't want to confuse future readers

Reply:
Done